### PR TITLE
pull request

### PR DESCRIPTION
**Pull request by msalamat**

### CODE REVIEW

File: create_box/main.py [hunk 1]
```diff
@@ -2,7 +2,13 @@
 
 
 def create_box(height, width, character):
-    pass
+    box = ''
+    for i in range(height): #loop from 0 to height 
+        box += (character * width) + '\n'
+        print (character * width ) #prints character "width" many times
```
Comment (santiagobasulto) [inline create_box/main.py]: Print? Don't print inside functions! (Unless it's specifically required)

File: create_box/main.py [hunk 2]
```diff
@@ -2,7 +2,13 @@
 
 
 def create_box(height, width, character):
-    pass
+    box = ''
+    for i in range(height): #loop from 0 to height 
+        box += (character * width) + '\n'
```
Comment (santiagobasulto) [inline create_box/main.py:6]: Good solution

**Comment by santiagobasulto** [comment]
Guys, this PR doesn't follow the standard of good names and description. Please fix.

**Comment by msalamat** [comment]
It is fixed now, updated with the extra challenge.

**Pull request closed without merging**